src/quo2/components/selectors/styles.cljs

```diff
@@ -1,130 +1,175 @@
-                        colors/neutral-80-opa-40))
-   :border-color     (if @checked?
-                       :none
-                       (get-color @checked? disabled? blurred-background?))})
+(def radio-background-color
+  (let [off      (colors/theme-colors colors/white-opa-40 colors/neutral-80-opa-40)
+        off-blur colors/white-opa-5]
+    {:normal {:enabled  off
+              :disabled (colors/alpha off 0.12)}
+     :blur   {:enabled  off-blur
+              :disabled (colors/theme-colors (colors/alpha off-blur 0.12)
+                                             (colors/alpha off-blur 0.015))}}))
 
-(defn radio
-  [checked? disabled? blurred-background?]
-  {:height           20
-   :width            20
-   :border-radius    20
-   :border-width     1
-   :border-color     (get-color checked?
-                                disabled?
-                                blurred-background?)
-   :background-color (when-not blurred-background?
-                       (colors/theme-colors colors/white
-                                            (colors/alpha colors/neutral-80
-                                                          0.4)))})
+(defn checkbox-background-color
+  [custom-color]
+  (let [on       (colors/custom-color-by-theme custom-color 50 60)
+        off      (colors/theme-colors colors/white-opa-40 colors/neutral-80-opa-40)
+        on-blur  (colors/theme-colors (colors/custom-color custom-color 50) colors/white)
+        off-blur colors/white-opa-5]
+    {:normal {:enabled  {:on  on
+                         :off off}
+              :disabled {:on  (colors/alpha on 0.3)
+                         :off (colors/alpha off 0.12)}}
+     :blur   {:enabled  {:on  on-blur
+                         :off off-blur}
+              :disabled {:on  (colors/alpha on-blur 0.3)
+                         :off (colors/theme-colors (colors/alpha off-blur 0.12)
+                                                   (colors/alpha off-blur 0.015))}}}))
+
+(def checkbox-border-color
+  (let [border     (colors/theme-colors colors/neutral-30 colors/neutral-70)
+        blur-light colors/neutral-80-opa-20
+        blur-dark  colors/white-opa-40]
+    {:normal {:enabled  border
+              :disabled (colors/alpha border 0.3)}
+     :blur   {:enabled  (colors/theme-colors blur-light blur-dark)
+              :disabled (colors/theme-colors (colors/alpha blur-light 0.06)
+                                             (colors/alpha blur-dark 0.12))}}))
+
+(def checkbox-prefill-background-color
+  (let [bg      (colors/theme-colors colors/neutral-30 colors/neutral-80)
+        bg-blur (colors/theme-colors colors/neutral-80-opa-10 colors/white-opa-10)]
+    {:normal {:enabled  bg
+              :disabled (colors/alpha bg 0.3)}
+     :blur   {:enabled  bg-blur
+              :disabled (colors/alpha bg-blur 0.03)}}))
+
+(defn- get-color
+  [color-map [& [disabled? blur? checked?]]]
+  (let [blur-kw     (if blur? :blur :normal)
```

Comment:
what is `-kw`? why not something like `blur-type` or something more expressive of what the variable is.

Reply:
-kw means keyword :sweat_smile:  but I can see it's not very clear.

It was hard to give a name to these vars, I replaced `-kw` by `-type`, it looks better. :+1: